docs/source/_static/css/navbar.css

```diff
@@ -0,0 +1,31 @@
+/* Introducing a new css file for handling navbar css */
```

Comment:
Introduced a new CSS file which will be the single source of truth for both `home.html`, `custom-title.html`, and `custom-navbar.html`.

This file will be expanded in the upcoming PR, I will shift the navbar CSS to this file